todo_app/view_model.py

```diff
@@ -0,0 +1,27 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+    
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def todo_items(self):
+        items_to_do = []
+        for myItem in self.items:
```

Comment:
Following the Python naming convention, the loop variable should be called `my_item`